src/common/attachments/uploadAttachmentsDialog.component.test.tsx

```diff
@@ -87,11 +87,32 @@ describe('Upload attachment dialog', () => {
       expect(screen.getByText('test1.txt')).toBeInTheDocument();
     });
 
+    await user.click(
+      await screen.findByRole('button', { name: 'Edit file test1.txt' })
+    );
+
+    expect(await screen.findByText('File name')).toBeInTheDocument();
+
+    // Checks if file extension is displayed. If it's editable, actual value will disappear after editing.
+    expect(await screen.findByText('.txt')).toBeInTheDocument();
+
+    const [name, title, description] = screen.getAllByRole('textbox');
+
+    await user.type(name, 'test.pdf');
+    await user.type(title, 'test title');
+    await user.type(description, 'test description');
```

Comment:
This approach using `getAllByRole('textbox')` is not ideal for accessibility. By relying only on the role and index, it becomes difficult to ensure that the correct input field is being interacted with.